include/wlr/types/wlr_input_method_v2.h

```diff
@@ -81,6 +103,8 @@ struct wlr_input_method_manager_v2 {
 	} events;
 };
 
+extern const struct wlr_surface_role input_popup_surface_v2_role;
```

Comment:
Can we move this somewhere else? It doesn't seem like this should be in the public header.

Seems to me that this could just be `static` inside `types/wlr_input_method_v2.c`.